vmdb/app/helpers/application_helper/dialogs.rb

```diff
@@ -95,7 +95,7 @@ def radio_options(field, url, value)
 
     extra_options = {
       :onclick  => remote_function(
-        :with     => "miqSerializeForm(this)",
+        :with     => "miqSerializeForm($(this).parent().attr('id'))",
```

Comment:
Mmm, good point. Not sure why I didn't just think to put the actual id here. Will change that.